packages/framework/agent-scheduler/src/test/tsconfig.json

```diff
@@ -1,14 +1,15 @@
 {
-	"extends": ["../../../../../common/build/build-common/tsconfig.test.json"],
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
+	"compilerOptions": {
+		"rootDir": "./",
+		"outDir": "../../lib/test",
+		"types": [],
```

Comment:
nit: can `types` be removed?

Reply:
The 'types: []' / 'noEmit: true' is a pattern we use for packages that contain only "type tests".  These are tests that invoke tsc and pass if the *.ts files successfully compile.  Nothing is executed.

The empty 'types: []' overrides the inherited 'types: ["node"]'.
The 'noEmit: true' tells TypeScript not to both producing files, since nothing is executed.